Gravity/Gravity/Extensions/RdoExtensions.cs

```diff
@@ -30,7 +30,10 @@ public static T ToHydratedDto<T>(this RDO rdo)
 
 					switch (fieldAttribute.FieldType)
 					{
-						case RdoFieldType.Currency:
+										case RdoFieldType.SingleObject:
```

Comment:
Looks like whitespace didn't quite come out right, but that's OK for now. Maybe we should finally get #18 done.